projects/1.18/assets/extreme-sound-muffler/extremesoundmuffler/lang/zh_cn.json

```
@@ -0,0 +1,57 @@
+{
+    "key.open_muffler_gui": "打开消音器界面",
+  
+    "inventory.btn": "消音器",
+  
+    "slider.btn.muffler.muffle": "开启消音",
+    "slider.btn.muffler.unmuffle": "关闭消音",
+    "slider.btn.play.play_sound": "播放/暂停音效",
+    "slider.btn.volume": "音量: ",
+  
+    "main_screen.btn.csl.recent": "最近",
+    "main_screen.btn.csl.all": "所有",
+    "main_screen.btn.csl.muffled": "已消音",
+    "main_screen.btn.csl.tooltip": "显示 %s 音效",
+    "main_screen.btn.tms.stop": "停止消音",
+    "main_screen.btn.tms.start": "开始消音",
+    "main_screen.btn.delete.sounds": "删除消音音效",
+    "main_screen.btn.delete.list": "删除消音列表",
+    "main_screen.btn.delete.anchor": "删除锚点",
+    "main_screen.btn.accept": "确定",
+    "main_screen.btn.cancel": "关闭",
+    "main_screen.btn.next_sounds": "下一个音效",
+    "main_screen.btn.previous_sounds": "上一个音效",
+    "main_screen.btn.anchors.disabled": "锚点已禁用",
+    "main_screen.btn.anchors.set_message": "首先设置锚点",
+    "main_screen.btn.anchors.set": "设置锚点",
+    "main_screen.btn.anchors.set_range": "范围: 1 - 32",
+    "main_screen.btn.anchors.set_title": "更改锚点标题",
+    "main_screen.btn.anchors.edit": "编辑锚点",
+  
+    "main_screen.main_title": "ESM - 主界面",
+    "main_screen.empty": "这里什么都没有..",
+    "main_screen.tip": "提示: %s",
+  
+    "main_screen.side_screen.x": "X: %s",
+    "main_screen.side_screen.y": "Y: %s",
+    "main_screen.side_screen.z": "Z: %s",
+    "main_screen.side_screen.radius": "范围半径: %s",
+    "main_screen.side_screen.dimension": "维度: %s",
+    "main_screen.side_screen.title": "标题: ",
+  
+    "tip.disable": "您可以在配置中禁用此提示",
+    "tip.change_volume": "你可以通过拖动滑块来更改消音的音量大小",
+    "tip.inv_button": "你可以通过在按钮上按住RMB并拖动它来移动物品栏按钮。",
+    "tip.inv_button_disable": "你可以在配置中禁用/启用物品栏界面上的消音器按钮",
+    "tip.play_sound": "你可以按相应的播放声音按钮来播放任何音效",
+    "tip.unmuffle": "你可以按“关闭消音”按钮来停止过滤所有选定的音效，再按一次就可以开启消音了",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
    "tip.unmuffle": "你可以按“关闭消音”按钮来停止过滤所有选定的音效，再按一次重新开启",
```
原文：**You can stop muffling all the selected sounds by pressing the Stop muffling sounds button, press it again to resume the muffling**